TGC/events/Germany VS Hungary Events.txt

```diff
@@ -0,0 +1,672 @@
+		623 = {  secede_province = GER }
+                3282 = {  secede_province = GER }
+                652 = { add_core = BAN }
+                653 = { add_core = BAN } 
+                3319 = { add_core = BAN }
+                
+                3309 = { add_core = BAN }
+                2538 = { add_core = BAN }
+                791 = { add_core = BAN }
+                792 = { add_core = BAN }
+                652 = {  secede_province = GER }
+                653 = {  secede_province = GER }
+                3319 = {  secede_province = GER }
+                
+                3309 = {  secede_province = GER }
+                2538 = {  secede_province = GER }
+                791 = {  secede_province = GER }
+                792 = {  secede_province = GER }
+                release = BAN
+                create_vassal = BAN
+                GER = { country_event = { id = 2000000002 days = 2 } }
+	}
+
+	option = {
+                name = "EVTOPTC2000000000"
+
+                set_global_flag = treaty_of_vienna_hungary
+
+		633 = { add_core = GER }
+		624 = { add_core = GER }
+		3286 = { add_core = GER }
+		623 = { add_core = GER }
+                3282 = { add_core = GER }
+                633 = {  secede_province = GER }
+		624 = {  secede_province = GER }
+		3286 = {  secede_province = GER }
+		623 = {  secede_province = GER }
+                3282 = {  secede_province = GER }
+                652 = { add_core = BAN }
+                653 = { add_core = BAN } 
+                3319 = { add_core = BAN }
+                
+                3309 = { add_core = BAN }
+                2538 = { add_core = BAN }
+                791 = { add_core = BAN }
+                792 = { add_core = BAN }
+                652 = {  secede_province = GER }
+                653 = {  secede_province = GER }
+                3319 = {  secede_province = GER }
+                
+                3309 = {  secede_province = GER }
+                2538 = {  secede_province = GER }
+                791 = {  secede_province = GER }
+                792 = {  secede_province = GER }
+                release = BAN
+                create_vassal = BAN
+                656 = {  secede_province = GER }
+                655 = {  secede_province = GER }
+                657 = {  secede_province = GER }
+                659 = {  secede_province = GER }
+                2533 = {  secede_province = GER }
+                660 = {  secede_province = GER }
+                661 = {  secede_province = GER }
+                654 = {  secede_province = GER }
+                3385 = {  secede_province = GER }
+                651 = {  secede_province = GER }
+                647 = {  secede_province = GER }
+                release = SIE
+                create_vassal = SIE
+                GER = { country_event = { id = 2000000002 days = 2 } }
+	        GER = { country_event = { id = 2000000004 days = 3 } }
+                badboy = 5
+	}
+        
+        option = {

```

Comment:
Also a separate discussion would be whether or not the Hohenzollerns would even want their Habsburg rivals to rule a sizable kingdom bordering their tenuously grasped domain, but for now I think we can make this work.

Reply:
You have to keep in mind that i wont be that sizeable anymore after the event chain (as it stands right now). But yes i see the problem, maybe use some other family (have that be an event that does political stuff maybe)

Reply:
Your explanation above about why it could be the Habsburgs actually makes sense and I don't mind it, but this is really only a matter of flavor text and not gameplay so it's not that important, what I meant here was merely that if (presumably) a different branch of the Habsburgs rules Austria and Bohemia as part of the new German Empire, also having Habsburgs in Hungary could prove to be a liability.

For example, if Germany loses a big war or gets disarmed Austria and Bohemia can become independent and Hungary can support them in fighting for their independence. But this is all a discussion of future potential, not pertaining to this pull request,